types/plugin/duration.d.ts

```diff
@@ -1,58 +1,61 @@
 import { PluginFunc } from 'dayjs'
 
 declare const plugin: PluginFunc
-export default plugin
+export as namespace plugin;
+export = plugin
 
-type DurationInputType = string | number | object
-type DurationAddType = number | object | Duration
+declare namespace plugin {
+  type DurationInputType = string | number | object
+  type DurationAddType = number | object | Duration
 
-export declare class Duration {
-  constructor (input: DurationInputType, unit?: string, locale?: string)
+  interface Duration {
```

Comment:
One small question, can we still use class Duration here, or is it necessary?

Reply:
An error occurred when using a class in a namespace, so the class was implemented as an interface.
It probably conflicts with the `const plugin` object. https://stackoverflow.com/a/51716899